### PR TITLE
ci: use renovate to upgrade Helm in ginkgo tests

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

File: .github/workflows/conformance-ginkgo.yaml [hunk 1]
```diff
@@ -265,6 +265,7 @@ jobs:
       - name: Install helm
         shell: bash
         run: |
+          # renovate: datasource=github-releases depName=helm/helm
           HELM_VERSION=3.7.0
           wget "https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz"
```
Comment (sayboras) [inline .github/workflows/conformance-ginkgo.yaml]: do we need to put v prefix + and remove v in wget command ?

Comment (nebril) [reply .github/workflows/conformance-ginkgo.yaml]: I think you're right!

**Review by sayboras — APPROVED** [review]
:ship:

**Review by brlbil — APPROVED** [review]

**Pull request merged**